modules/ROOT/pages/rest-sdk/edits/rest-sdk-connectivity.adoc

```diff
@@ -0,0 +1,53 @@
+= REST SDK
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+REST SDK enables you to create a connector from an API specification (OAS or RAML)
+and uses a metadata-driven approach to accelerate connector development. REST SDK
+provides a quick and relevant connectivity experience that is easy to use and requires
+little to no configuration.
+
+With REST SDK you can:
+
+* Enable instant connectivity via metadata
+* Reduce the speed needed to build a new connector from several months to days or minutes
+* Provide a unified connector designer that welcomes non-developer users
+* Eliminate the need to generate and maintain system specific code and static assets
+
+== How to Use REST SDK
+
+Use the following documentation as a step-by-step guide to build your own
+connector using REST SDK.
+
+* xref:rest-sdk/features.adoc[Overview]
++
+Learn about different use cases and features that REST SDK provides.
+
+* xref:rest-sdk/project-setup.adoc[Setup and Installation]
++
+Ensure you meet the prerequisites and install the REST SDK plugin.
+
+* xref:rest-sdk/scaffold-connector-descriptor.adoc[Scaffold a Connector Descriptor]
++
+Scaffold a connector descriptor from an API specification.
+
+* xref:rest-sdk/create-connector.adoc[Create a Connector]
++
+Create a connector from a connector descriptor and an API specification.
+
+* xref:rest-sdk/customization-layer.adoc[Create a Customization Layer]
```

Comment:
I have a technical question about the relationship between this optional task and the task that follows it: do you not have to create a customization layer in order to customize the connector? Or is Customize Your Connector also optional?
If this is a question that your audience is not likely to have, then just ignore me. I'm sure that the text in the full .adoc for each clarifies this, but it might not be a bad idea to clarify this relationship in the intro .adoc.

Reply:
Yup, you do not have to create a customization layer to customize your connector. You can customize your connector with just a connector descriptor or a connector descriptor + optional customization layer